### PR TITLE
Neobauer - fixes for 0,1 and new solution for task 2

### LINKED ISSUE #2: 0. Add Cargo and Ship classes from classes.
powinno być gotowe do 23:00 15.06.2020

### LINKED ISSUE #6: 1. Task: Cargo (operator==)
W klasie Cargo napisz operator porównania (operator==), który będzie sprawdzał, czy towary są takie same.

### LINKED ISSUE #7: 2. Task: Cargo (getter, constructor)
Do klasy Cargo dopisz gettery oraz odpowiedni konstruktor, który wypełni wszystkie pola tej klasy.

### PR DESCRIPTION
**Pull request by NeoBauer**

### CODE REVIEW

**Review by TomaszEichler — APPROVED** [review]
Now ok :)

File: shm/cargo.hpp [hunk 1]
```diff
@@ -2,15 +2,19 @@
 #include <string>
 class Cargo {
```
Comment (mateusz-talma) [inline shm/cargo.hpp:3]: add empty line

File: shm/cargo.hpp [hunk 2]
```diff
@@ -2,15 +2,19 @@
...
 public:
+    Cargo(std::string name, size_t amount, size_t basePrice);
     Cargo& operator+=(size_t amount);
     Cargo& operator-=(size_t amount);
     bool operator==(std::string name);
+    std::string GetName() const;
+    size_t GetAmount() const;
+    size_t GetBasePrice() const;
```
Comment (mateusz-talma) [inline shm/cargo.hpp:14]: too many empty lines. One line is enough

**Review by mateusz-talma — CHANGES_REQUESTED** [review]
some little changes

File: shm/cargo.hpp (hunk 2 above)
Comment (TomaszEichler) [reply shm/cargo.hpp:14]: we can autoformat it later, he uses vim

File: shm/cargo.hpp (hunk 1 above)
Comment (TomaszEichler) [reply shm/cargo.hpp:3]: we can autoformat it later, he uses vim

File: shm/cargo.hpp (hunk 2 above)
Comment (mateusz-talma) [reply shm/cargo.hpp:14]: ok :)

**Review by mateusz-talma — APPROVED** [review]
Remember about code formatting :)

**Pull request merged**